### ISSUE TITLE
preview error

### ISSUE THREAD

**Issue by fancybaby** [kartik-v/bootstrap-fileinput#1441]
I limited the size of files, when I choose images with some of them are oversized and some are not, and I upload after selected, all the images' state would be changed to success in preview thumbnail  after upload success, has anyone ever had this problem?

**Comment by angarc** [comment]
I'm also having a preview error, but I can't even get my images to show. I keep getting a javascript error that says `p is undefined`. 

I'm loading the image previews in html, not with javascript, althought I've found that I get the same error with javascript.

**Comment by fancybaby** [comment]
Which version did you try? maybe you can try the other version fileinput.js and fileinput.css.

**Comment by fancybaby** [comment]
> I'm also having a preview error, but I can't even get my images to show. I keep getting a javascript error that says `p is undefined`.
> 
> I'm loading the image previews in html, not with javascript, althought I've found that I get the same error with javascript.

Which version did you try? maybe you can try the other version fileinput.js and fileinput.css.

**Comment by angarc** [comment]
Yeah, I was able to solve it. 

Turns out that all the link tags in the installation section of the documentation, despite a couple of syntax errors, also have some other bugs. When I imported the javascript files using webpacker, things started working correctly. 

However, I'm still not able to import styles (scss) with webpacker as there are some url errors in the scss code (or at least there is some miscommunication between webpacker and the scss files). So I just solved this by including the link the to stylesheet in my view. 

It's a really wonky setup and definitely not in accordance with best practices, but at least it works. I raised an issue about it and the owner placed an "enhancement" badge on it. Hopefully we see an update.

Also, now that everything works, I'm not getting the preview error that you're describing... I wonder what's going on.
